ui/src/app/views/homepage/component.html

```diff
@@ -10,11 +10,13 @@
         </div>
       </div>
       <div class="row" style="flex-direction: row; align-items: flex-start; justify-content: space-between; flex-wrap: nowrap;">
-        <div class="card add-card" (click)="addWorkspace()">
-          <div class="card-text">
+        <div role="tooltip" aria-haspopup="true" class="card add-card tooltip tooltip-md tooltip-bottom-left"
```

Comment:
There's also a link in the sidebar we should disable: https://github.com/all-of-us/workbench/blob/9b60ccc97eb838477fb31c203b8b48aea9becdbf/ui/src/app/views/signed-in/component.html#L44